recipes/flake8/meta.yaml

```diff
@@ -1,63 +0,0 @@
-{%set name = "flake8" %}
```

Comment:
@sannykr You should probably rebase your commits to the latest master and force-push to this branch.